app/routes/bdb/components/BdbDetail.js

```diff
@@ -185,6 +185,7 @@ export default class BdbDetail extends Component<Props, State> {
     const events =
       companyEvents &&
       companyEvents
+        .slice()
```

Comment:
for future reference, lodash methods usually don't mutate, so they're often good alternatives (`sort` and `slice`)